build.sbt

```diff
@@ -282,4 +284,125 @@ val generate = taskKey[Unit]("generate code from APIs")
 // Library versions
 val jacksonVersion = "2.17.0"
 val openApiToolsJacksonBindNullableVersion = "0.2.6"
-val log4jVersion = "2.23.1"
\ No newline at end of file
+val log4jVersion = "2.23.1"
+
+/*
+ ********************
+ * Release settings *
+ ********************
+ */
+import ReleaseTransformations._
+
+lazy val skipReleaseSettings = Seq(
```

Comment:
add skipReleaseSettings to CLI
add javaOnlyReleaseSettings to server and client

Reply:
Done. What about apiDocs ?